newTab.html

```diff
@@ -4,7 +4,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<html>
+<html manifest="newtab.appcache">
```

Comment:
As above. Maybe we should drop this altogether... not sure how appcache and Service Workers interact.